src/main/resources/config.yml

```diff
@@ -1,5 +1,9 @@
 # The storage types the plugin uses
 
+# There are two types of storage: SQLite and MySQL.
+connection-type: SQLite
```

Comment:
Best to use all lowercase for connection type, although you do change it to uppercase later.